go-controller/pkg/config/config.go

```diff
@@ -760,6 +762,11 @@ var OVNGatewayFlags = []cli.Flag{
 		Usage:       "Setup nodeport based ingress on gateways.",
 		Destination: &cliConfig.Gateway.NodeportEnable,
 	},
+	&cli.BoolFlag{
+		Name:        "disable-snat",
```

Comment:
can we call it disable-snat-ecmp-egress or disable-snat-multiple-gws or something like that? The pods in a namespace that do not have multiple gateways should still be snat'ed.

Reply:
done